src/features/content-answers/__tests__/InlineError.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { render, screen } from '@testing-library/react';
+
+import InlineError from '../InlineError';
+
+import messages from '../messages';
+
+jest.mock('react-intl', () => jest.requireActual('react-intl'));
+
+describe('features/content-answers/InlineError', () => {
+    const IntlWrapper = ({ children }: { children?: React.ReactNode }) => {
+        return <IntlProvider locale="en">{children}</IntlProvider>;
+    };
```

Comment:
does this work:
```
jest.mock('react-intl', () => ({
    ...jest.requireActual('react-intl'),
    FormattedMessage: ({ defaultMessage }: { defaultMessage: string }) => <span>{defaultMessage}</span>,
}));
```

or does it still need the whole wrapper? when does the console error occur? why does a test like `ContentAnalyticsErrorState.test.tsx` not need it?

Reply:
yeah I think this works. The console error occurs when I only mocks 'react-intl'. The 
test ContentAnalyticsErrorState.test.tsx did not need it because it was testing the element instead of the text.